### DISCUSSION TITLE
Make CL mod incompatible with all lazer only mods/mod customisation

### DISCUSSION THREAD

**Discussion by TextAdventurer12** [ppy/osu#25734] [Ideas]
If the idea of the CL mod is to act as a stable emulator, then it should be 1:1 with scores doable on the stable client. This also makes maintaining legacy scoreboards easier, since they can just be filtered by the CL mod, and there is no need for a special 'stable' mod/tag.

**Comment by bdach** [comment]
> and there is no need for a special 'stable' mod/tag

where is this even coming from i was pretty sure this was not ever going to be a thing

**Reply by TextAdventurer12** [reply]
I saw it thrown around on the discord server when talking about preserving legacy scores and handling the scoring algorithm with the two slightly different rulesets

**Reply by Givikap120** [reply]
There's internal IsLegacy flag in score, so game knows what it's doing
Why should user care about it if it's the same ruleset.
If you eager to know is it lazer scores or not - you can click on the score. If it has new statistics such as hit sliderends - it's a lazer score.
